### ISSUE TITLE
Currently broken Plugins

### ISSUE THREAD

**Issue by HypedDomi** [Strencher/BetterDiscordStuff#324]
Here is a List of currently known broken Plugins:
- [BetterBannedUsers](https://github.com/Strencher/BetterDiscordStuff/tree/master/BetterBannedUsers)
- [ChannelDms](https://github.com/Strencher/BetterDiscordStuff/tree/master/ChannelDms)
- [Copier](https://github.com/Strencher/BetterDiscordStuff/tree/master/Copier)
- [PronounDB](https://github.com/Strencher/BetterDiscordStuff/tree/master/PronounDB)
- [ShowAllActivities](https://github.com/Strencher/BetterDiscordStuff/tree/master/ShowAllActivities)
- [ShowSessions](https://github.com/Strencher/BetterDiscordStuff/tree/master/ShowSessions)
- [StatusEverywhere](https://github.com/Strencher/BetterDiscordStuff/tree/master/StatusEverywhere)
- [SuppressReplyMentions](https://github.com/Strencher/BetterDiscordStuff/tree/master/SuppressReplyMentions)
- [UserDetails](https://github.com/Strencher/BetterDiscordStuff/tree/master/UserDetails)
- [VoiceChatNotifications](https://github.com/Strencher/BetterDiscordStuff/tree/master/VoiceChatNotifications)
- [VoiceUsersCounter](https://github.com/Strencher/BetterDiscordStuff/tree/master/VoiceUsersCounter)

*Please ping me when you found a plugin which isn't in this list*

**Comment by HistoricTheater** [comment]
@HypedDomi 
What level of breaking? #297 is broken as it stops working seconds/minutes/hours depending on your servers activity, but it will load in to discord and work until those conditions are met. It is not working how it worked before or as intended.

**Comment by HypedDomi** [comment]
I mean Plugins which don't compile or completely stopped working. PlatformIndicators main functionality works, that's why it isn't in the list. I may make a list of all known bugs soon

**Comment by Eden333** [comment]
I loved having "ShowAllActivities" :(

**Comment by foursplaysroblox** [comment]
Unread Count badges is broken
![image](https://user-images.githubusercontent.com/82078139/208021808-e202771c-7e44-452c-9c79-599083b5d135.png)

**Comment by HypedDomi** [comment]
> Unread Count badges is broken

Updated the list 👍

**Comment by PitchBlackNights** [comment]
@HypedDomi 

TypingUsersPlus is broken
![image](https://user-images.githubusercontent.com/79918522/208145997-684d7abe-994f-4944-8af1-f7783185c4e4.png)

**Comment by HypedDomi** [comment]
> TypingUsersPlus is broken
> 

Thank you. Added it to the list

**Comment by stewcop23** [comment]
WhoReacted is broken
![image](https://user-images.githubusercontent.com/37968460/208185965-7d0a5bfb-9cf9-45f0-bfc9-c91619ad76b0.png)

**Comment by HypedDomi** [comment]
Isnt a plugin from strencher

**Comment by Rayjacker** [comment]
ShowAllActivities has been fixed, so can be removed from the list or put into a Fixed column

**Comment by B-Nerd** [comment]
so what is the problem that you can't or won't fix

**Comment by HypedDomi** [comment]
> so what is the problem that you can't or won't fix

Strencher is mostly working on other things rn and has no time or motivation to fix the plugins

**Comment by rollingmoai** [comment]
Copier is crashing

**Comment by HypedDomi** [comment]
> Copier is crashing

Thats an issue with BD, not with Copier. Wait for a BD Update

**Comment by B-Nerd** [comment]
> > so what is the problem that you can't or won't fix
> 
> Strencher is mostly working on other things rn and has no time or motivation to fix the plugins

it is late should there not be someone to takeover or help because it has not been updated for 7 months

**Comment by HypedDomi** [comment]
> it is late should there not be someone to takeover or help because it has not been updated for 7 months

We're always open for PR's

**Comment by rollingmoai** [comment]
ShowAllActivites could not be compiled
![image](https://user-images.githubusercontent.com/75471378/227676951-59156da3-1efb-4a64-80f2-52e436c774da.png)

**Comment by fowlis** [comment]
> > > so what is the problem that you can't or won't fix
> > 
> > 
> > Strencher is mostly working on other things rn and has no time or motivation to fix the plugins
> 
> it is late should there not be someone to takeover or help because it has not been updated for 7 months

Here's an idea: you do it. You're not paying anyone to fix it, so why not do it yourself? Nobody needs to fix their plugins to satisfy you - plugin devs do it as a hobby, and do it unpaid.

**Comment by MCHAMSTERYT2** [comment]
When will you update pronoundb?

**Comment by HypedDomi** [comment]
> When will you update pronoundb?

There is no ETA for it

**Comment by HistoricTheater** [comment]
@HypedDomi 
#297 is not fixed, as previously reported, status icons disappear anytime someone higher up in  the memberlist login/logout
Tested on stable and ptb

**Comment by HypedDomi** [comment]
> @HypedDomi #297 is not fixed, as previously reported, status icons disappear anytime someone higher up in the memberlist login/logout Tested on stable and ptb

Can you try if you still have this error in the latest version? (1.4.2)

**Comment by Horsion** [comment]
I've just checked for #297 on version 1.4.2 and I still get this error with the status icons disappearing when someone goes offline then online.

**Comment by Lennervh** [comment]
any solutions for the failed to compile error?

**Comment by B-Nerd** [comment]
> any solutions for the failed to compile error?

the only solution to this problem for how the project looks is to fork it this is a dead project

**Comment by domi-btnr** [comment]
> > any solutions for the failed to compile error?
> 
> the only solution to this problem for how the project looks is to fork it this is a dead project

It's not dead. Strencher just doesn't have enough time and/or motivation for fixing Plugins. But if you know how to fix it, you can fork it and make a PR

**Comment by B-Nerd** [comment]
> > > any solutions for the failed to compile error?
> > 
> > 
> > the only solution to this problem for how the project looks is to fork it this is a dead project
> 
> It's not dead. Strencher just doesn't have enough time and/or motivation for fixing Plugins. But if you know how to fix it, you can fork it and make a PR

it has not been fixed in a year some of the plugins are 2 years so those plugins are a very dead & your fork part that is why i typed it the creator lost time/motivation for this fixing plugins so a fork is the only way we will get then fixed

**Comment by MooMcMooman** [comment]
@domi-btnr ShowAllActivities is broken, causing a crash whenever you click on a profile.

**Comment by RoseIlanga** [comment]
BetterUserDates appears to be broken too. It loads, but isn't showing the timestamp.

**Comment by domi-btnr** [comment]
> @domi-btnr ShowAllActivities is broken, causing a crash whenever you click on a profile.

Fixed

**Comment by OPADILOP** [comment]
Any update on fixing these plugins?

**Comment by domi-btnr** [comment]
> Any update on fixing these plugins?

Strencher does it when he has the time and motivation

**Comment by shamshitty** [comment]
dunno if this issue thread still gets looked at but a bug ive recently noticed with BetterUserDates is that only the date for joining discord gets the timestamp, meanwhile the date for joining a server doesnt
![image](https://github.com/Strencher/BetterDiscordStuff/assets/68004605/d5bc0e84-15b0-4ec6-91a6-d09f3dd85cff)
@domi-btnr

**Comment by domi-btnr** [comment]
> dunno if this issue thread still gets looked at but a bug ive recently noticed with BetterUserDates is that only the date for joining discord gets the timestamp, meanwhile the date for joining a server doesnt ![image](https://private-user-images.githubusercontent.com/68004605/328708003-d5bc0e84-15b0-4ec6-91a6-d09f3dd85cff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxMjQ4NzksIm5iZiI6MTcxNTEyNDU3OSwicGF0aCI6Ii82ODAwNDYwNS8zMjg3MDgwMDMtZDViYzBlODQtMTViMC00ZWM2LTkxYTYtZDA5ZjNkZDg1Y2ZmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA3VDIzMjkzOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZhMzY3ZTI4YTZhZDNkMTk1OGE0NDc2YmZkN2RkZGQzOWQ0NDJiYmVjYTY4Zjk4NmZhNDE4MDAxNTg5MmM4NzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.fHaFREbNtjlVDQ-S4kKtBZ5sK7_puxoiGmVhtpBSp70) @domi-btnr

Yeah, I look at it whenever I get a notification. I'll add it to the list

**Comment by shamshitty** [comment]
lil update, now neither dates get the timestamp

**Comment by domi-btnr** [comment]
> lil update, now neither dates get the timestamp

Updated the issue. Thank you

**Comment by zrodevkaan** [comment]
> lil update, now neither dates get the timestamp

Would you have any logs for me to look at ?

**Comment by RickGotTaken** [comment]
As of your recent update to InvisibleTyping, it has not been working.
https://github.com/user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3

**Comment by domi-btnr** [comment]
> As of your recent update to InvisibleTyping, it has not been working. [user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3](https://github.com/user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3)

It worked fine when I tested it and it also works fine for others. Could you please provide a full size screen recording and make sure you’re on the latest version which is 1.3.5 atm

**Comment by RickGotTaken** [comment]
> > As of your recent update to InvisibleTyping, it has not been working. [user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3](https://github.com/user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3)
> 
> It worked fine when I tested it and it also works fine for others. Could you please provide a full size screen recording and make sure you’re on the latest version which is 1.3.5 atm

Do you mean 1.3.4? Since I can't seem to update.

**Comment by domi-btnr** [comment]
> > > As of your recent update to InvisibleTyping, it has not been working. [user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3](https://github.com/user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3)
> > 
> > 
> > It worked fine when I tested it and it also works fine for others. Could you please provide a full size screen recording and make sure you’re on the latest version which is 1.3.5 atm
> 
> Do you mean 1.3.4? Since I can't seem to update.

Nope. 1.3.5. You need to downloade the latest file on GitHub or you wait till the update is approved

**Comment by RickGotTaken** [comment]
> > > > As of your recent update to InvisibleTyping, it has not been working. [user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3](https://github.com/user-attachments/assets/b08e3067-f671-4d73-a319-6bb12b8d43d3)
> > > 
> > > 
> > > It worked fine when I tested it and it also works fine for others. Could you please provide a full size screen recording and make sure you’re on the latest version which is 1.3.5 atm
> > 
> > 
> > Do you mean 1.3.4? Since I can't seem to update.
> 
> Nope. 1.3.5. You need to downloade the latest file on GitHub or you wait till the update is approved

The plugin is working for me as of version 1.3.5.

**Comment by MagmaBro123** [comment]
This issue has been open for 2 years...

Are these plugins actually being updated?

**Comment by zrodevkaan** [comment]
> This issue has been open for 2 years...
> 
> Are these plugins actually being updated?

No.

**Comment by MagmaBro123** [comment]
> > This issue has been open for 2 years...
> > Are these plugins actually being updated?
> 
> No.
> 
> Me and Domi have been working on plugins here and there. Other then that nope

Then please remove them from the BD site. If they don't work then they shouldn't be on there.

**Comment by zrodevkaan** [comment]
You tell me this like I can do anything about it.

**Comment by Riddim-GLiTCH** [comment]
> > > This issue has been open for 2 years...
> > > Are these plugins actually being updated?
> > 
> > 
> > No.
> > Me and Domi have been working on plugins here and there. Other then that nope
> 
> Then please remove them from the BD site. If they don't work then they shouldn't be on there.

This isn't exactly up to the maintainers of the repo. The BD site is managed by the BD staff, and it is up to *them* to decide whether or not an abandoned plugin should be removed.

**Comment by Huderon** [comment]
> Sure, the developer could ask for their plugins to be removed, but it would still be in the control of the BD staff.

That's not the case. Devs can remove their addons whenever they want

**Comment by Kionaaa** [comment]
BetterFolders, Message Logger, Member Count, are broken too.

**Comment by Riddim-GLiTCH** [comment]
> BetterFolders, Message Logger, Member Count, are broken too.

None of these are strenchers plugins. Go report issues in their respective repos, not here.

**Comment by Riddim-GLiTCH** [comment]
A manager of this repo should probably lock the comments tbh

**Comment by domi-btnr** [comment]
Since the majority of Plugins is archived and the repo is in "maintenance only" mode, this Issue will be closed
